libevmasm/Assembly.h

```diff
@@ -154,6 +154,12 @@ class Assembly
 		bool _includeSourceList = true
 	) const;
 
+	/// Loads the JSON representation of assembly.
+	/// @param _json JSON object containing assembly
+	/// @param _loadSources true, if source list should be included, false otherwise.
+	/// @returns true on success, false otherwise
+	static std::shared_ptr<Assembly> loadFromAssemblyJSON(Json::Value const& _json, std::vector<std::string> const& _sourceList = {}, bool _isCreation = true);
```

Comment:
I think that `Assembly::fromJSON()` would be pretty clear as a name and shorter:
```suggestion
	static std::shared_ptr<Assembly> fromJSON(Json::Value const& _json, std::vector<std::string> const& _sourceList = {}, bool _isCreation = true);
```